deploy/kubernetes/dolphinscheduler/templates/statefulset-dolphinscheduler-master.yaml

```diff
@@ -109,11 +109,9 @@ spec:
           volumeMounts:
             - mountPath: "/opt/dolphinscheduler/logs"
               name: {{ include "dolphinscheduler.fullname" . }}-master
-            {{- include "dolphinscheduler.sharedStorage.volumeMount" . | nindent 12 }}
             - name: config-volume
               mountPath: /opt/dolphinscheduler/conf/common.properties
               subPath: common.properties
-            {{- include "dolphinscheduler.etcd.ssl.volumeMount" . | nindent 12 }}
```

Comment:
Why delete the ssl volume of etcd?

ref: #13924   
@qingwli PTAL.

Reply:
it seems this is also needed in `api`, 
so I kept this and adjust the order to make it clear to read.